src/index.css

```diff
@@ -149,7 +143,6 @@ body {
   color: var(--cpd-color-text-primary);
   color-scheme: dark;
   margin: 0;
-  font-family: var(--font-family);
```

Comment:
Can we just remove the font-family from the body css? 
where is `--cpd-font-family-sans` used to set the computed font family?

Reply:
okay wasnt awar of this. Thanks for clarifying.